src/globals/_charts.scss

```diff
@@ -1,4 +1,9 @@
-@import '~@carbon/charts/styles.css';
+@import '~@carbon/charts/styles/styles.scss';
+@import './vars';
+
+.#{$iot-prefix}--card .#{$prefix}--chart-holder {
```

Comment:
This should be moved to the `_card.scss` partial, no?

Reply:
agreed. Will address in other PR